src/dynsight/_internal/trajectory/trajectory.py

```diff
@@ -37,16 +55,21 @@ def spatial_average(
         selection: str = "all",
         num_processes: int = 1,
     ) -> Insight:
+        """Average the descripotor over the neighboring particles."""
```

Comment:
Going to be annoying here, but I think the documentation should state what meta comes with the Insight